Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Diagnostics/DiagnosticDescriptors.cs

```diff
@@ -145,5 +145,13 @@ public static class DiagnosticDescriptors
             category: "AWSLambdaCSharpGenerator",
             DiagnosticSeverity.Error,
             isEnabledByDefault: true);
+
+        public static readonly DiagnosticDescriptor MultipleConfigureMethodsNotAllowed = new DiagnosticDescriptor(
+            id: "AWSLambda0119",
+            title: "Conflicting Service Configuration Methods Detected",
+            messageFormat: "Both '{0}' and '{1}' methods are present. Only {1} will be used. The {0} method will be ignored. Please consolidate your service configuration into {1}.",
+            category: "AWSLambdaCSharpGenerator",
+            DiagnosticSeverity.Error,
```

Comment:
Since it is an `Error`, which I think is the right level, then the assembly fails to compile so the wording "The {0} method will be ignored." isn't accurate because the build failed.

Reply:
makes sense